### PR TITLE
Make this repo as `massa-proto-rs`

### PR DESCRIPTION
**Pull request by aoudiamoncef**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Copyright (c) 2023 MASSA LABS <info@massa.net>
 
 [package]
-name = "massa_proto_bindings"
+name = "massa_proto_rs"
 version = "0.1.0"
 edition = "2021"
 description = "Protobuf rust bindings for the Massa blockchain"
-repository = "https://github.com/massalabs/massa-proto-bindings/"
+repository = "https://github.com/massalabs/massa-proto/"
```
Comment (bilboquet) [inline Cargo.toml:8]: missing a `-rs` at the end of the url, isn't it?

Comment (aoudiamoncef) [reply Cargo.toml:8]: Good catch

**Review by bilboquet — COMMENTED** [review]
I've pushed a fix, please review it
if it's ok for you I think we can merge

File: build.rs [hunk 2]
```diff
@@ -15,8 +15,7 @@ mod tonic {
 
     /// This function is responsible for building the Massa protobuf
     pub fn build() -> Result<(), Box<dyn std::error::Error>> {
-        let current_dir = env::current_dir()?;
-        let parent_dir = current_dir.parent().ok_or("no parent directory found")?;
+        let parent_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
```
Comment (aoudiamoncef) [inline build.rs:18]: Good catch

**Comment by aoudiamoncef** [comment]
@bilboquet Thanks for you fix 👍🏻

**Pull request merged**